dex/src/test/scala/com/wavesplatform/dex/tools/ConfigCheckerCliSpec.scala

```diff
@@ -0,0 +1,72 @@
+package com.wavesplatform.dex.tools
+
+import com.typesafe.config.ConfigValueFactory
+import com.wavesplatform.dex.settings.BaseSettingsSpecification
+import com.wavesplatform.dex.tool.ConfigChecker
+import org.scalatest.EitherValues
+import org.scalatest.matchers.should.Matchers
+
+class ConfigCheckerCliSpec
+    extends BaseSettingsSpecification
+    with Matchers
+    with EitherValues {
+
+  "ConfigChecker" should "match fully config sample " in {
```

Comment:
> "match fully config sample "

as I remember there was an issue with tests those have spaces on edges. Could you remove excessive space after "sample" and in other tests too?